stl/inc/yvals_core.h

```diff
@@ -458,6 +458,13 @@
 #define _CONSTEXPR17 inline
 #endif // _HAS_CXX17
 
+// C++20 constexpr additions
+#if _HAS_CXX20
+#define _CONSTEXPR20 constexpr
+#else // ^^^ has C++20 constexpr additions / no C++20 constexpr additions vvv

```

Comment:
I know this is the existing language but do we really want to say 'additions'? I've seen that in some old Dinkumware stuff but I've never quite understood what that meant...

How about `// ^^^ constexpr in C++20 and later // inline in C++17 and earlier vvv` ? (And similarly fix up the C++17 one above)

/cc @CaseyCarter @StephanTLavavej @barcharcraz

Reply:
"additions" makes sense to me: we use this macro to annotate functions that exist in earlier standard revisions (without `constexpr`), and had `constexpr` "added" to their declarations in C++20. Having said that, I think your suggestion more clearly and directly says what we want to say.

Reply:
I can push the following change if you like it:

```cpp
// N4842 [dcl.constexpr]/1: "A function or static data member declared with the
// constexpr or consteval specifier is implicitly an inline function or variable"

// Functions that became constexpr in C++17
#if _HAS_CXX17
#define _CONSTEXPR17 constexpr
#else // ^^^ constexpr in C++17 and later / inline (not constexpr) in C++14 vvv
#define _CONSTEXPR17 inline
#endif // ^^^ inline (not constexpr) in C++14 ^^^

// Functions that became constexpr in C++20
#if _HAS_CXX20
#define _CONSTEXPR20 constexpr
#else // ^^^ constexpr in C++20 and later / inline (not constexpr) in C++17 and earlier vvv
#define _CONSTEXPR20 inline
#endif // ^^^ inline (not constexpr) in C++17 and earlier ^^^
```

Reply:
LGTM

Reply:
+1